project/ParadoxSupport.scala

```diff
@@ -24,8 +22,13 @@ object ParadoxSupport {
       val log = streams.value.log
       val classpath = (fullClasspath in Compile).value.files.map(_.toURI.toURL).toArray
       val classloader = new java.net.URLClassLoader(classpath, this.getClass().getClassLoader())
-      lazy val scanner = new FastClasspathScanner("akka").addClassLoader(classloader).scan()
-      val allClasses = scanner.getNamesOfAllClasses.asScala.toVector
+      import _root_.io.github.classgraph.ClassGraph
+      lazy val scanner = new ClassGraph()
+        .whitelistPackages("akka")
+        .verbose()
+        .addClassLoader(classloader)
+        .scan()
+      val allClasses = scanner.getAllClasses.getNames.asScala.toVector
```

Comment:
is it the same thing that has moved, or what is the background?

Reply:
The FastClassPathScananer is replaced by the ClassGraph, and only ClassGraph is being maintained now